adapter/database.go

```diff
@@ -42,13 +42,17 @@ func DumpDDLs(d Database) (string, error) {
 	return strings.Join(ddls, ";\n\n"), nil
 }
 
-func RunDDLs(d Database, ddls []string) error {
+func RunDDLs(d Database, ddls []string, isSafety bool) error {
 	transaction, err := d.DB().Begin()
 	if err != nil {
 		return err
 	}
 	fmt.Println("-- Apply --")
 	for _, ddl := range ddls {
+		if isSafety && strings.Contains(ddl, "DROP") {
+			fmt.Printf("Not executed: %s;\n", ddl)
```

Comment:
sqldef's output is designed to always be a valid input for `mysql`/`psql`. Could you prefix `--` to make it a SQL comment?

And if you follow https://github.com/k0kubun/sqldef/pull/44/files#r339372757, it'd be `-- Skipped: %s;`.

Reply:
I understand.
I think `-Skipped:% s;` is good.